src/build/resources.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+const fs = require("fs").promises;
+const Viz = require("viz.js");
+const vizRenderOpts = require("viz.js/full.render.js");
+
+const COPY_FILES_PATTERN = /\.(jpg|jpeg|png|gif|ms)/;
+const VIZ_RENDER_PATTERN = /\.(dot|neato|fdp|sfdp|twopi|circo)/;
+
+async function buildResources(pageIndex, buildOpts) {
```

Comment:
This function replaces the gulp tasks which used to copy images and build diagrams. It uses localized page URLs as the destination directories.